pkg/filter/regex.go

```diff
@@ -29,6 +29,14 @@ func (f *RegexpFilter) Filter(response *ffuf.Response) (bool, error) {
 	}
 	matchdata := []byte(matchheaders)
 	matchdata = append(matchdata, response.Data...)
+	// is the raw value a fuzzing keyword
+	if _, fuzzed := inputs[f.valueRaw]; fuzzed {
+		matched, err := regexp.Match(string(inputs[f.valueRaw]), matchdata)
```

Comment:
We need to use `regexp.QuoteMeta` for the input data in order to properly escape all the regex control characters as the input data might have them.

Reply:
Tried searching such a method with no luck, good to know!

Reply:
It is now fixed.